keyboards/rabbit/rabbit68/readme.md

```diff
@@ -0,0 +1,20 @@
+Rabbit 68
+===
+
+![Rabbit 68 Project](https://github.com/kaiec/rabbit68)
+
+A compact 68 keys ortholinear keyboard.
+
+Keyboard Maintainer: [Kai Eckert](https://github.com/kaiec)  
+Hardware Supported: Rabbit68 PCB rev1  
+Hardware Availability: Open Source 
+
+Make example for this keyboard (after setting up your build environment):
+
+    make rabbit/rabbit68:default
+
+Install examples:
+
+    make rabbit/rabbit68:default:teensy
```

Comment:
Could you use the universal `flash` command here, instead?  

```suggestion
    make rabbit/rabbit68:default:flash
```

It's not necessary, but it's nicer.

Reply:
Sure, didn't know about it. Thanks both @fauxpark and @drashna for the feedback!